README.md

```diff
@@ -1,20 +1,36 @@
 # Meilix
 
-[![Join the chat at https://gitter.im/fossasia/meilix](https://badges.gitter.im/fossasia/meilix.svg)](https://gitter.im/fossasia/meilix?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![Build Status](https://travis-ci.org/fossasia/meilix.svg?branch=master)](https://travis-ci.org/fossasia/meilix)
+[![Join the chat at https://gitter.im/fossasia/meilix](https://badges.gitter.im/fossasia/meilix.svg)](https://gitter.im/fossasia/meilix?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Build Status](https://travis-ci.org/fossasia/meilix.svg?branch=master)](https://travis-ci.org/fossasia/meilix)
 
 Beautiful Linux OS for Hotels and Public Event Kiosks
 * lxqt as the standard Desktop Environment
 * based on ubuntu/debian architecture
 * Philosophy: light weight, fast, customized
 * reasonable preconfigured settings for the use case
-* system lock
+* [system lock](https://github.com/fossasia/meilix-systemlock/)
 
-## Starting Development
+# Feature
```

Comment:
@meets2tarun I don't think feature is the right word to use we can use scripts maybe